cast/tests/utils/prepare_for_tests.sh

```diff
@@ -5,20 +5,32 @@ if ! which starknet-devnet > /dev/null 2>&1; then
   exit 1
 fi
 
```

Comment:
> I know this was here already but I guess we can add it here it's just one line

Context matters. You can do `set -e`, but then `|| exit 1` are to be removed, because they are no longer needed. up to you if you prefer one way or another.

Reply:
You are right, I think `set -e` is better (less boilerplate).